src/main/resources/application.conf

```diff
@@ -0,0 +1,35 @@
+s3 {
+    endpoint = "https://s3.eu-west-2.amazonaws.com/"
+}
+
+sqs {
+    endpoint = "https://sqs.eu-west-2.amazonaws.com"
+    queue {
+        input = "https://sqs.eu-west-2.amazonaws.com/set_me_account_number/tdr-file-format-intg"
+        input = ${?INPUT_QUEUE}
+        fileformat = "https://sqs.eu-west-2.amazonaws.com/set_me_account_number/tdr-file-format-intg"
+        fileformat = ${?FILE_FORMAT_QUEUE}
```

Comment:
Do these config values need defaults? It doesn't look like the default would do anything. Would it be better to just have `input = ${INPUT_QUEUE}`, etc.? That way it should fail with an obvious error message if you forget to set a value when running it locally rather than calling an endpoint that doesn't exist.

Reply:
Yeah, fair point. I've taken them out.

Reply:
Do you need to take the `?` out too? I don't know what happens if you include without anything to fall back on.

Reply:
If they're not set and there's no question mark you get an error but that's what we want. I've taken out the question mark.